components/calendar/src/calendar-input/calendar-input.js

```diff
@@ -110,9 +127,12 @@ export const CalendarInput = ({
                     onFocus={onFocus}
                     value={date}
                     onChange={editable ? handleChange : undefined}
-                    validationText={error || warning}
-                    error={!!error}
-                    warning={!!warning}
+                    validationText={
+                        pickerResults.errorMessage ||
```

Comment:
@kabaros @tomzemp I think it would be better if we removed the default "Pick a date" label and instead allow the consumer component to pass a custom label if needed. This change would solve the current issue where InputFields that don't require a label must explicitly pass `label=""` to remove the default "Pick a date" text.
What do you think?

Reply:
In principle, I'd say that we should keep "Pick a date" in case people are already consuming the component, so they do not update and find that the label has changed, but of our core apps there is maybe only Data Entry (Beta) which is using the default label. Possibly (but not super likely), there are third-party apps using the default label with this component.

I'd be fine with removing the "Pick a date" if there's no objections from @kabaros